styles/style.css

```diff
@@ -56,16 +56,20 @@ a:active {
     font-weight: 400;
 }
 
-/* .left-menu,
-.right-side-bar {
-    position: sticky;
-    top: 0;
-
-    z-index: 1;
-} */
-
 main {
+    height: 100vh;
+
     background-color: var(--second-color);
+
+    overflow-y: scroll;
+    /* Hide scrollbar */
+    -ms-overflow-style: none;  /* IE and Edge */
+    scrollbar-width: none;  /* Firefox */
+}
+
+main::-webkit-scrollbar {
+    /* Hides scrollbar for Chrome, Safari and Opera */
+    display: none;
```

Comment:
It was interesting to find that I need different lines of code, each one implementing the same behaviour on different browsers. Need to find if there is a shorthand to hide scrollbars on every browser.

Reply:
You get an add-on that takes your css and adds code for different browsers. When you use SASS which get's compiled, it automatically adds code for other browsers. So must be an add on for normal css

Reply:
Thank you. I'll defenitely have a read about SASS in the near future.